packages/rest-api/src/utils/fetchBridgeTransaction.ts

```
@@ -0,0 +1,61 @@
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ query: graphqlQuery }),
+  })
+
+  const graphqlData = await graphqlResponse.json()
+
+  if (
+    graphqlData.data &&
+    graphqlData.data.bridgeTransactions &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)